cluster/gce/config-test.sh

```diff
@@ -135,3 +135,6 @@ NETWORK_PROVIDER="${NETWORK_PROVIDER:-none}" # opencontrail
 OPENCONTRAIL_TAG="${OPENCONTRAIL_TAG:-R2.20}"
 OPENCONTRAIL_KUBERNETES_TAG="${OPENCONTRAIL_KUBERNETES_TAG:-master}"
 OPENCONTRAIL_PUBLIC_SUBNET="${OPENCONTRAIL_PUBLIC_SUBNET:-10.1.0.0/16}"
+
+# Optional: if set to true, kube-up will configure the cluster to run e2e tests.
+ENABLE_E2E_TEST_ENVIRONMENT=${KUBE_ENABLE_E2E_TEST_ENVIRONMENT:-true}
```

Comment:
As per other review comments, the default should be false, even for test evironments, and should need to be explicitly overridden for specialised storage test environments.